src/System.Runtime.Numerics/tests/ComplexTests.cs

```diff
@@ -292,6 +292,7 @@ public static IEnumerable<object[]> ACos_Advanced_TestData()
         }
 
         [Theory, MemberData("ACos_Advanced_TestData")]
+        [SkipOnTargetFramework(TargetFrameworkMonikers.NetFramework)]
```

Comment:
I think all of the tests in this class should be checked as well; they don't seem to be testing anything platform-specific.